### PR TITLE
Vault CA integration prototype

### PR DESCRIPTION
**Pull request by lei-tang**
Vault CA integration prototype

### CODE REVIEW

File: security/pkg/util/vaultutil.go [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+	saReqJSON, err := json.Marshal(saReq)
+	if err != nil {
+		return nil, fmt.Errorf("failed to marshal the service account review request: %v", err)
+	}
+	req, err := http.NewRequest("POST", k8sAPIServerURL, bytes.NewBuffer(saReqJSON))
+	if err != nil {
+		return nil, fmt.Errorf("failed to create a HTTP request: %v", err)
+	}
+	req.Header.Set("Content-Type", "application/json")
+	req.Header.Set("Authorization", "Bearer "+reviewerToken)
```
Comment (wattli) [inline security/pkg/util/vaultutil.go]: Is this part of the call credential to let ApiServer accept your request?

Comment (wattli) [reply security/pkg/util/vaultutil.go]: If so, this is also a jwt, right?

Comment (lei-tang) [reply security/pkg/util/vaultutil.go]: The answer is yes to both of your questions.

**Pull request merged**